allennlp/data/dataset_readers/quora_paraphrase.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict
+import re
+import logging
+import csv
+import zipfile
+import io
+
+from overrides import overrides
+
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.fields import LabelField, TextField, Field
+from allennlp.data.instance import Instance
+from allennlp.data.tokenizers import Tokenizer, WordTokenizer
+from allennlp.data.tokenizers.word_splitter import JustSpacesWordSplitter
+from allennlp.data.token_indexers import TokenIndexer, SingleIdTokenIndexer
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def parse_file_uri(uri: str):
+    match = re.fullmatch(r'\((.*)\)#(.*)', uri)      # pylint: disable=anomalous-backslash-in-string
+    if match and len(match.groups()) == 2:
+        return match.groups()[0], match.groups()[1]
+    else:
+        return uri, None
+
+@DatasetReader.register("quora_paraphrase")
+class QuoraParaphraseDatasetReader(DatasetReader):
+    """
+    Reads a Quora paraphrase data
```

Comment:
An explanation of the expected data format here would be nice (e.g., this is a TSV file, with the following columns).